samples/bot-consent-azuread/csharp/Microsoft.Teams.Samples.Bot.Consent.AzureAD/Manifest/manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.5/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.13",
```

Comment:
I believe this is only applicable for Teams. Could you please help to change it back to 1.12?

Ref: https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-sdk?tabs=javascript%2Cmanifest-teams-toolkit
<img width="435" alt="image" src="https://user-images.githubusercontent.com/31851992/173278553-acc8d130-604f-4d19-b83a-98119015339d.png">

Reply:
@scottperham has sorted this Wajeed